src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -244,6 +243,9 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
     # Directory to place new units.
     new_corpus_dir = self._create_temp_corpus_dir('new')
 
+    # Directory to place merged corpus.
+    merge_corpus_dir = self._create_merge_corpus_dir()
```

Comment:
also, why do we need to create it here instead of leaving the creation inside `_merge_new_units`? I see the exception in the PR description but don't understand what's the difference